pkg/monitor/monitor/template.go

```diff
@@ -240,11 +240,11 @@ func buildAddressRelabelConfigByComponent(kind string) *config.RelabelConfig {
 		return &config.RelabelConfig{
 			Action:      config.RelabelReplace,
 			Regex:       addressPattern,
-			Replacement: "$1.$2-importer.$3:$4",
+			Replacement: "$2.$3:$4",
```

Comment:
Please revert the change to this file, after changing the instance label, this should work.

Reply:
reverted